Networking/NetworkingTests/Remote/RemoteTests.swift

```diff
@@ -423,28 +423,32 @@ final class RemoteTests: XCTestCase {
         XCTAssertNotNil(result)
     }
 
-    /// Verifies that WordPressOrg request parses WordPressApiError
+    /// Verifies that RESTRequest request doesn't parse WordPressApiError
     ///
-    func test_wordpress_org_request_parses_wordpress_api_error() async throws {
+    func test_wordpress_org_request_does_not_parse_wordpress_api_error() async throws {
```

Comment:
I don't understand this test case, should the request parse the error though?

Reply:
The original test case validated that `WordPressApiValidator` is used in `WordPressOrgRequest`. 

In `RESTRequest` we decided to use `PlaceholderDataValidator` which will not validate the request. So, I updated the test case to check that no errors are parsed. 
i.e. `test_wordpress_org_request_does_not_parse_wordpress_api_error` and `test_wordpress_org_request_does_not_parse_dotcom_error` will ensure that `PlaceholderDataValidator` is used.